test/markdown-admonition.json

```diff
@@ -0,0 +1,21 @@
+{
+  "default": false,
+  "max-one-sentence-per-line": {
+    "ignored_words": [
+      "ie",
+      "i.e",
+      "eg",
+      "e.g",
+      "etc",
+      "ex",
+      "!!",
+      "??"
```

Comment:
this is the work around

Reply:
Is it two characters instead of three on purpose?

Reply:
yes, because the logic takes a look at the element in front of a line ending, (line ending is defined by another regex) and compares those, not including the matched regex. as the third exclamation mark is part of the regex, we only need two here

Reply:
Thanks for the explanation! It would be great to get this workaround

Reply:
I added this to my `.markdownlint.yaml` and it works. Thanks!